pkg/runner/runner.go

```diff
@@ -169,8 +169,6 @@ func ReportSubcommandHandler(userInput input.Input, mutex *sync.Mutex,
 	subdomains = input.CreateSubdomains(userInput.ReportWordSub, protocolTemp, urlUtils.CleanProtocol(target))
 
 	if userInput.ReportSubdomainDB {
-		sonar := opendb.SonarSubdomains(urlUtils.CleanProtocol(target), false)
```

Comment:
Can you please comment these two lines instead of deleting them? Just comment them and add a comment saying `service not working`